content/en/docs/ambient/usage/extend-waypoint-wasm/index.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Distributing WebAssembly Modules
```

Comment:
The title was supposed to be "Extend waypoints with WebAssembly plugins" ?

Reply:
corrected